driver_cpl/bld/namelist_files/namelist_defaults_drv.xml

```diff
@@ -4,6 +4,275 @@
 
 <namelist_defaults>
 
+<!-- =========================================  -->
```

Comment:
Yes. The driver and data components namelists are now generated with the python nmlgen class which is much more robust and understandable than the previous build-namelist. As part of this we introduced a new schema combining namelist definition and namelists defaults into one file  in cime_config/namelist_definition_drv.xml. If you determine what differences there were between the acme cime5.0 and namelist_defaults.xml and namelists_definition.xml and what you are trying to merge in now - I am happy to help incorporate those into the new namelist_definition_drv.xml. My guess as that there will only be minor changes that need to be brought in.